src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/switches/SwitchPortConfigSpec.groovy

```diff
@@ -91,13 +91,13 @@ class SwitchPortConfigSpec extends HealthCheckSpecification {
 
         when: "Bring port down on the switch"
         def port = topology.getAllowedPortsForSwitch(sw).find { "LINK_DOWN" in northbound.getPort(sw.dpId, it).state }
-        antiflap.portDown(sw.dpId, port)
+        northbound.portDown(sw.dpId, port)
 
         then: "Port is really DOWN"
         Wrappers.wait(WAIT_OFFSET) { assert "PORT_DOWN" in northbound.getPort(sw.dpId, port).config }
 
         when: "Bring port up on the switch"
-        def portUp = antiflap.portUp(sw.dpId, port)
+        def portUp = northbound.portUp(sw.dpId, port)
```

Comment:
cleanup should be fixed as well. Also don't really understand why this change is required